modules/persistent-storage-csi-olm-operator-install.adoc

```diff
@@ -2,12 +2,29 @@
 //
 // * storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc
 // * storage/container_storage_interface/osd-persistent-storage-csi-aws-efs.adoc
+// * storage/persistent_storage/rosa-persistent-storage-aws-efs-csi.adoc
+// * storage/container_storage_interface/persistent-storage-csi-smb-cifs.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="persistent-storage-csi-olm-operator-install_{context}"]
 = Installing the {FeatureName} CSI Driver Operator
 
-The link:https://github.com/openshift/aws-efs-csi-driver-operator[AWS EFS CSI Driver Operator] (a Red Hat operator) is not installed in {product-title} by default. Use the following procedure to install and configure the {FeatureName} CSI Driver Operator in your cluster.
+The {FeatureName} CSI Driver Operator (a Red Hat operator) is not installed in {product-title} by default. Use the following procedure to install and configure the {FeatureName} CSI Driver Operator in your cluster.
```

Comment:
Also per the [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces) there should be a non breaking space in between Red and Hat:
```suggestion
The {FeatureName} CSI Driver Operator (a Red{nbsp}Hat operator) is not installed in {product-title} by default. Use the following procedure to install and configure the {FeatureName} CSI Driver Operator in your cluster.
```